utils/scripts/ocm/ocm.py

```diff
@@ -1405,6 +1437,28 @@ def oc_login_as_admin(self):
     )
     dbaaspolicy_parser.set_defaults(func=ocm_obj.update_dbaaspolicy_config)
 
+    # Argument Parser to install RHODA for given version from Dev Catalog
+    install_rhoda_dev_parser = subparsers.add_parser(
+        "install_rhoda_dev",
+        help=("To Install RHODA from Develop Catalog Image"),
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
```

Comment:
What is this first section for?

Reply:
@ikanias This is for Passing arguments to the method **_install_rhoda_dev_** specified on line 1460 **_install_rhoda_dev_parser.set_defaults(func=ocm_obj.install_rhoda_dev)_** 

We started with defining the argument parser to invoke method **_install_rhoda_dev_** on line 1441 to 1445. We are adding helper text and defining the class for the argument. So when we invoke ocm,py with the argument as **_./utils/scripts/ocm/ocm.py _install_rhoda_dev_**, the control passed here to verify the additional arguments required to invoke this method, in this case **--rhoda-version** and **--rhobs-token**